CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ if(MIOPEN_ENABLE_SQLITE)
     pkg_check_modules(SQLITE3 REQUIRED sqlite3)
 endif()
 find_package(BZip2)
-find_package(nlohmann_json 3.11.2 REQUIRED)
+find_package(nlohmann_json 3.9.1 REQUIRED)
```

Comment:
I am Ok with this, but just in case -- why this change is required?

Reply:
So we won't have to update the image. It already contains 3.9.1

Reply:
You need to link with `nlohmann_json::nlohmann_json` so the include paths are correctly added.